documentation/advanced/tutorial-i18n-localization.asciidoc

```diff
@@ -215,11 +216,11 @@ public class MainLayout extends VerticalLayout implements LocaleChangeObserver {
 }
 ----
 
-==== Frontend projects
+== Frontend Projects
 
 For frontend applications only, to set the RTL mode you can call `document.dir = 'rtl'`.
 
-==== Adding RTL support to your custom elements
+== Adding RTL Support to Your Custom Elements
 
 If you have your own custom elements, there are a few steps needed in order to
```

Comment:
Should this be rewritten a bit to indicate that not only may custom web components need tweaks, but any custom UI feature (e.g. css) as well?

Reply:
Something like this?

```suggestion
If you have your own custom elements or your application has custom styles,
there are a few steps needed in order to add RTL support to them:
```

And also changing the title to **Adding RTL Support to Your Custom Elements or Application**

Reply:
maybe "or your application has custom styles or **layouts**"?
of course layouts would be css-based too, so not sure how meaningful it is to make that distinction...

Reply:
I would say that "custom styles" covers _layouts_, in this case.